schemas/blocks_schema.json

```diff
@@ -76,53 +80,91 @@
           "additionalProperties": false
         }
       },
-      "drops": {
+      "states": {
         "type": "array",
         "items": {
-          "type": "object",
-          "properties": {
-            "minCount": {
-              "type": "number",
-              "minimum": 0,
-              "description": "minimum number or chance, default : 1"
-            },
-            "maxCount": {
-              "type": "number",
-              "minimum": 0,
-              "description": "maximum number or chance, default : minCount"
+            "type": "object",
+            "properties": {
+              "name": {
+                "description": "The name of the property",
+                "type": "string"
+              },
+              "type": {
+                "description": "The type of the property",
+                "type": "string"
+              },
+              "values": {
+                "description": "The possible values of the property",
+                "type": "array"
+              },
+              "num_values": {
+                "description": "The number of possible values",
+                "type": "number",
+                "minimum": 1
+              }
+              
             },
-            "drop": {
-              "oneOf": [
-                {
-                  "type": "integer",
-                  "minimum": 0
-                },
-                {
-                  "type": "object",
-                  "properties": {
-                    "id": {
-                      "type": "integer",
-                      "minimum": 0
-                    },
-                    "metadata": {
-                      "type": "integer",
-                      "minimum": 0
-                    }
-                  },
-                  "required": [
-                    "id",
-                    "metadata"
-                  ],
-                  "additionalProperties": false
-                }
-              ]
-            }
-          },
-          "required": [
-            "drop"
-          ],
-          "additionalProperties": false
-        }
+            "required": [
+              "name",
+              "type",
+              "num_values"
+            ],
+            "additionalProperties": false
+          }
+      },
+      "drops": {
```

Comment:
why did you change that ?

Reply:
Change what? Not really sure what' you're talking about here

Reply:
The drop schema

Reply:
I just made it possible for the drop array to contain strings for textIds, not really sure why github is saying I changed that much, maybe it's a spacing problem

Reply:
Ok that can be solved by using integer ids

Reply:
So block drops should go back to using the block id?